### PR TITLE
Added table of contents to readme

### PR DESCRIPTION
**Pull request by jherrflexion**
# Description
Added table of contents to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,473 +1,233 @@
 # CDC Trusted Intermediary
 
-This document provides instructions for setting up the environment, running the application, and performing various tasks such as compiling, testing, and contributing to the project.
+This document provides comprehensive instructions for setting up, running, and contributing to the project.
+
+---
+
+## Table of Contents
+
+1. [Requirements](#requirements)
```
Comment (basiliskus) [inline README.md:9]: Is there a way to auto-generate this table? Otherwise we'll need to maintain it when we update add/remove sections

**Review by basiliskus — COMMENTED** [review]
I need to spend more time reviewing later, but it seems a few sections are missing. I noticed the RS setup section is gone

File: README.md (hunk 1 above)
Comment (jherrflexion) [reply README.md:9]: I see what I did - on it. Will also investigate the auto generation

Comment (saquino0827) [reply README.md:9]: One thing for auto generating ToC. If you use VS code, there's a plugin called 'Markdown - All In One' that has a ToC generator. I'm trying it out for our script read me.

Comment (halprin) [reply README.md:9]: There is a built-in ToC in GitHub.  It is automatically generated based on the headings.  It is accessed by clicking on the button on the right hand side of the ReadMe.  See this image...

image

**Review by luis-pabon-tf — APPROVED** [review]
All sections are in the readme and it works good.

**Review by tjohnson7021 — APPROVED** [review]
Love this!

**Pull request merged**